Stripe/STPAPIRequest.m

```diff
@@ -65,20 +79,34 @@ + (NSURLSessionDataTask *)getWithAPIClient:(STPAPIClient *)apiClient
 + (void)parseResponse:(NSURLResponse *)response
                  body:(NSData *)body
                 error:(NSError *)error
-           serializer:(id<STPAPIResponseDecodable>)serializer
+          serializers:(NSArray<id<STPAPIResponseDecodable>>*)serializers
            completion:(STPAPIResponseBlock)completion {
-
-    NSDictionary *jsonDictionary = body ? [NSJSONSerialization JSONObjectWithData:body options:(NSJSONReadingOptions)kNilOptions error:NULL] : nil;
-    id<STPAPIResponseDecodable> responseObject = [[serializer class] decodedObjectFromAPIResponse:jsonDictionary];
-    NSError *returnedError = [NSError stp_errorFromStripeResponse:jsonDictionary] ?: error;
-    if ((!responseObject || ![response isKindOfClass:[NSHTTPURLResponse class]]) && !returnedError) {
-        returnedError = [NSError stp_genericFailedToParseResponseError];
-    }
-
+    id<STPAPIResponseDecodable> responseObject;
+    NSError *returnedError;
     NSHTTPURLResponse *httpResponse;
     if ([response isKindOfClass:[NSHTTPURLResponse class]]) {
         httpResponse = (NSHTTPURLResponse *)response;
     }
+
+    if (serializers.count == 0) {
+        returnedError = [NSError stp_genericFailedToParseResponseError];
+    } else {
+        NSDictionary *jsonDictionary = body ? [NSJSONSerialization JSONObjectWithData:body options:(NSJSONReadingOptions)kNilOptions error:NULL] : nil;
+        NSString *object = jsonDictionary[@"object"];
+        Class serializerClass = [serializers.firstObject class];
+        for (id<STPAPIResponseDecodable> serializer in serializers) {
```

Comment:
This logic is a little weird. If none of the stripeObject keys match, it will end up with the firstObject's class. Is that intentional?